tests/Test/AbstractFixerTestCase.php

```diff
@@ -703,6 +706,34 @@ private static function assertValidDescription(string $fixerName, string $descri
         self::assertFalse(strpos($descriptionType, '``'), \sprintf('[%s] The %s must no contain sequential backticks.', $fixerName, $descriptionType));
     }
 
+    private static function assertOptionDefault(FixerOptionInterface $option, FixerInterface $fixer): void
+    {
+        if (!$option->hasDefault()) {
+            return;
+        }
+
+        $allowedValues = $option->getAllowedValues();
+
+        if (!is_countable($allowedValues) || 1 !== \count($allowedValues)) {
+            return;
+        }
+
+        $allowedValueSubset = reset($allowedValues);
```

Comment:
```suggestion
        $allowedValueSubset = $allowedValues[0];
```

Reply:
My first version had `$allowedValues[0]`, but:
```
 ------ --------------------------------------
  Line   tests/Test/AbstractFixerTestCase.php
 ------ --------------------------------------
  721    Offset 0 does not exist on null.
         🪪  offsetAccess.notFound
 ------ --------------------------------------
```

Reply:
as you check on null [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8529/commits/7e80a95be7e28675753487ebb1ac5b04138c1651) , it will complain no more

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8530 will allow this change.

Reply:
> as you check on null [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8529/commits/7e80a95be7e28675753487ebb1ac5b04138c1651) , it will complain no more

After fixing the check to `===` it is all god.

Reply:
Nope, it still [complains](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/14010938576/job/39230639672?pr=8529) as `$allowedValues` might be an empty array according do PHPDoc, https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8530 is addressing this issue.